.circleci/deploy.sh

```diff
@@ -12,11 +12,12 @@ export PAPERTRAIL_SYSTEM=$3
 # Set the right place for paper trail logging
 cat .ebextensions/02_papertrail.config | envsubst '$PAPERTRAIL_HOST:$PAPERTRAIL_PORT:$PAPERTRAIL_SYSTEM' >tmp
 mv tmp .ebextensions/02_papertrail.config
+cat .circleci/Dockerrun.aws.json.template | envsubst > Dockerrun.aws.json
```

Comment:
I moved this over to the deployment script from the circleCI setup file - it fits here better